docker-compose.yml

```diff
@@ -225,6 +228,12 @@ services:
       - "5432"
     ports:
       - "${TEST_POSTGRES_PORT}:5432"
+  presto:
+    image: starburstdata/presto:302-e.2 
```

Comment:
Any particular reason to use [starburstdata/presto](https://hub.docker.com/r/starburstdata/presto) instead of [prestosql/presto](https://hub.docker.com/r/prestosql/presto)?